### PR TITLE
[#OSF-7436]Change icons on top right of Project Overview page to make them more universal

### PR DESCRIPTION
**Pull request by chennan47**
## Purpose
Change icons on top right of Project Overview page to make them more universal

## QA notes:
1. create a new project and make it public
2. go to the project overview page
Before change:
screen shot 2017-03-01 at 1 20 05 pm
If you click the duplicate button, it shows
screen shot 2017-03-01 at 1 20 51 pm
if you click the share button, it shows
screen shot 2017-03-01 at 1 22 51 pm

After change:
screen shot 2017-03-01 at 1 21 28 pm
If you click the new fork button, it shows
screen shot 2017-03-01 at 1 22 07 pm
If you click the "duplicate" button, it shows
screen shot 2017-03-01 at 1 23 49 pm

## Ticket
https://openscience.atlassian.net/browse/OSF-7436

### CODE REVIEW

File: website/templates/project/project.mako [hunk 1]
```diff
@@ -49,7 +49,53 @@
...
+                        % if not user_name:
+                            data-bind="tooltip: {title: 'Log in or create an account to duplicate this project', placement: 'top'}"
+                        % endif
+                        >
+                            <div class="dropdown">
+                                <a
+                                % if user_name:
+                                    class="btn btn-default dropdown-toggle" data-toggle="dropdown" type="button" aria-expanded="false"
+##                                     data-bind="tooltip: {title: 'Duplicate', placement: 'bottom', container : 'body'}"
+##                                     data-target="#duplicateModal" data-toggle="modal"
```
Comment (brianjgeiger) [inline website/templates/project/project.mako]: Are these commented out? They should be deleted instead.

File: website/templates/project/project_base.mako [hunk 2]
```diff
@@ -27,9 +27,6 @@
     <%include file="retracted_registration.mako" args="node='${node}'"/>
 % else:
     ${next.body()}
-##  % if node['node_type'] == 'project':
```
Comment (brianjgeiger) [inline website/templates/project/project_base.mako]: I guess I didn't see these comments. Please remove as well.

File: website/templates/project/project_base.mako [hunk 3]
```diff
@@ -27,9 +27,6 @@
...
-        <%include file="modal_duplicate.mako"/>
-##  % endif
```
Comment (brianjgeiger) [inline website/templates/project/project_base.mako]: Same: please remove.

File: website/templates/project/project_base.mako (hunk 2 above)
Comment (chennan47) [inline website/templates/project/project_base.mako]: It is removed. So it is marked red background.

File: website/templates/project/project_base.mako (hunk 3 above)
Comment (chennan47) [inline website/templates/project/project_base.mako]: It is removed. So it is marked red background.

**Pull request merged**